tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf

```diff
@@ -11,46 +11,63 @@ Object.Pipeline {
 	mixout-playback.0 {
 		index 2
 		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.1.playback'
+			stream_name 'copier.HDA.2.1'
 		}
 
 		Object.Widget.copier.1 {
 			stream_name 'Analog Playback and Capture'
 			dai_type "HDA"
 			copier_type "HDA"
 		}
+		Object.Widget.gain.1 {
+			Object.Control.mixer.1 {
+				name 'Main PGA playback 2'
```

Comment:
on mixout paths, the gain is local to the path, it's not a Main PGA at all.

We should only use Main on the dai pipelines (after mixing).

Likewise on capture if we split a path the Main gain should be in the dai pipeline prior to the demux.

Essentially we should use 'Main' only when changing the value of the gain has an impact of all sources/sinks connected to a dai.